lldb/include/lldb/Target/ThreadPlanStack.h

```diff
@@ -194,7 +230,23 @@ class ThreadPlanStackMap {
 
 private:
   Process &m_process;
-  using PlansList = std::unordered_map<lldb::tid_t, ThreadPlanStack>;
+  // We don't want to make copies of these ThreadPlanStacks, there needs to be
```

Comment:
If you're using shared pointers, I'm not sure why we can't store them directly in the unordered map. The map would only ever be copying shared_pointers around, which won't touch the underlying object.